corehq/apps/reports/models.py

```diff
@@ -760,6 +760,9 @@ def send(self):
             self.delete()
             return
 
+        if self.owner.is_deleted():
+            return
```

Comment:
it means we don't have to check those reports every single time they fire, which is more than just removing them from the DB.

how does one even delete and undelete a web user? or is this a mobile worker?